Library/Formula/mobiledevice.rb

```diff
@@ -0,0 +1,14 @@
+class Mobiledevice < Formula
+  homepage "https://github.com/imkira/mobiledevice"
+  url "https://github.com/imkira/mobiledevice/archive/v2.0.0.tar.gz"
+  sha256 "07b167f6103175c5eba726fd590266bf6461b18244d34ef6d05a51fc4871e424"
+
+  def install
+    system "make", "install", "CC=#{ENV.cc}", "PREFIX=#{prefix}"
+  end
+
+  test do
+    has_version = shell_output("#{bin}/mobiledevice version").strip
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!